www/base/src/app/schedulers/schedulers.controller.coffee

```diff
@@ -2,3 +2,23 @@ class schedulers extends Controller
     constructor: ($log, $scope, $location, dataService) ->
         data = dataService.open().closeOnDestroy($scope)
         $scope.schedulers = data.getSchedulers()
+        $scope.enabledValues = []
+
+        # Assumes an order.
+        $scope.schedulers.onNew = (s) ->
+            $scope.enabledValues.push(s.enabled)
+
+        $scope.change = ->
+            i = 0
+            while i<$scope.enabledValues.length
+                oldValue = $scope.enabledValues[i]
+                scheduler = $scope.schedulers[i]
+                newValue = scheduler.enabled
```

Comment:
looks too complicated.
why dont you pass the scheduler id in the change callback?